### DISCUSSION TITLE
Language problem

### DISCUSSION THREAD

**Discussion by 20i173yi** [David-Maisonave/MediaFileDuplicateFinder#6] [Ideas]
"Hey, I want to translate the buttons in the GUI window to another language, but I can't seem to find the .resx file. Is it hard-coded in there?"

**Comment by David-Maisonave** [comment]
The code uses [Avalonia UI](https://avaloniaui.net/) xaml files to store GUI text, and I haven't seen an auto translator for these types of files.
I'm not the original developer, otherwise I would have program it using a *.resx file. There are existing auto-translators for RESX files.

At one time I tried refactoring the code, but it turn out to be too much work, and I gave up on it.

Sorry.
